mozci/configuration.py

```diff
@@ -79,6 +82,10 @@ def __init__(self, cache_config):
         super_config.setdefault("stores", {"null": {"driver": "null"}})
         super(CustomCacheManager, self).__init__(super_config)
 
+        for store, conf in self._config["stores"].items():
+            if store != "null":
+                logger.debug(f"Active cache store {store} with conf {conf}")
```

Comment:
I guess you were planning to remove this?

Reply:
Well no, I mean that's up to debate.

My idea here is to leave this enabled for now and see if the log line is displayed on Taskcluster.

It may be pretty hard to be sure that the cache is working otherwise  as we don't have access to the docker-worker cache directly.